test/performance/specs/post-editor.spec.js

```diff
@@ -659,7 +659,7 @@ test.describe( 'Post Editor Performance', () => {
 
 				const startTime = performance.now();
 
-				await page.getByText( 'Test' ).click();
+				await page.getByRole( 'tab', { name: 'Test' } ).click();
```

Comment:
@ntsekouras I'm curious why you changed this? It's causing the performance tests to fail for 6.5, where this had the role of a button.

Reply:
I think there were two elements with the `test` and we wanted that tab one.